server/routes/BookRoute.js

```diff
@@ -59,6 +59,24 @@ router.post('/', async (req, res) => {
   }
 });
 
+// @desc    Get one book's details
+// @route   GET /book/details/:id
+// @access  Public
+router.get('/details/:isbn', async (req, res) => {
+  let { isbn } = req.params;
+
+  try {
+    let first = await axios.get(`https://www.googleapis.com/books/v1/volumes?q=isbn:${isbn}`);
+    console.log(first.data.items[0].id);
+    let response = await axios.get(
+      `https://www.googleapis.com/books/v1/volumes/${first.data.items[0].id}`
+    );
+    res.send(response.data);
+  } catch (error) {
+    console.error(`Could not complete Google Books API request for isbn ${isbn}: `, error);
```

Comment:
Should send an error in response along with the console log

Reply:
resolved